sample-compose/app/src/main/java/co/nimblehq/sample/compose/ui/AppDestination.kt

```diff
@@ -16,7 +17,7 @@ sealed class AppDestination(val route: String = "") {
 
     open var parcelableArgument: Pair<String, Any?> = "" to null
 
-    object Up : AppDestination()
+    data class Up(val results: List<Result> = emptyList()) : AppDestination()
```

Comment:
```suggestion
    data class Up(val results: Map<String, Any> = emptyList()) : AppDestination()
```
Why don't we use `Map` instead? 🤔

Reply:
@lydiasama good suggestion, fixed in https://github.com/nimblehq/android-templates/pull/530/commits/54ef3e3b4700b5aaa04f10fb134876ce54c3ac48